src/main/pages/user-management/organizations.adoc

```diff
@@ -64,77 +81,120 @@ CHE_MAIL_SMTP_AUTH_USERNAME=no-reply@gmail.com
+[IMPORTANT]
+====
+This action can not be reverted and all workspaces created under the organization will be deleted.
+All members of the organization will receive an email notification to inform about the deletion of the organization. Only system adminstrators or administrators of the organization delete an organization or sub-organization. 
+====
 
 [id="organization-resources"]
-== Organization Resources
+== Allocating resources for organizations
 
-Workspaces created in an organization use the resources allocated by the system administrator to that organization. Resources for sub-organizations are taken from the parent organization. Admins can control whether all, or a portion of the parent’s resources are available to the sub-organization.
+The workspace use the resources of the organization that is allocated by the system administrator. The resources for sub-organizations are taken from the parent organization. Administrators control which resources of the parent organization are available to the sub-organization.
 
 [id="managing-limits"]
 == Managing Limits
 
-*Action restricted to*: Eclipse Che system administrator and admins of the organization.
+The system configuration defines the default limits. The administrator of the organization manages only the limits of its sub-organizations. No resource limits applies to the organization by default.  The following are the limits defined by the system adminstrator:
+
+* *Workspace Cap*: The maximum number of workspaces that exists in the organization. +
+* *Running Workspace Cap*: The maximum number of workspaces that run simultaneously in the organization. +
+* *Workspace RAM Cap*: The maximum amount of RAM that a workspace uses in GB.
 
-The organization default caps are taken from the system configuration. The admin of the organization can manage only the limits of it’s sub-organizations. By default, there are no resource limits applied to the organization so all members can benefit from all the allocated resources. If an organization admin wishes to set limits they have three options:
+[NOTE]
+====
+Managing limits is restricted to the Eclipse Che system administrator and administrator of the organization.
+====
 
-* *Workspace Cap*: The maximum number of workspaces that can exist in the organization. +
-* *Running Workspace Cap*: The maximum number of workspaces which can run simultaneously in the organization. +
-* *Workspace RAM Cap*: The maximum total RAM organization workspaces can use in GB.
 
 [id="update-organization-and-sub-organization-member-roles"]
-== Update Organization and Sub-Organization Member Roles
+== Updating organization and sub-organization member roles
+
+To edit the member role:
+
+. Click *Edit* button in the "Actions" column. You update the role of the selected member in the pop-up window. 
 
-*Action restricted to*: Eclipse Che system administrator and admins of the organization.
+. Click *Save* to confirm the update.
+
+[NOTE]
```

Comment:
Likewise, I'd move this admonition to the beginning of this section for the same reason.